conf/exactmatch-val.bess

```diff
@@ -0,0 +1,111 @@
+# vim: syntax=py
```

Comment:
should this file and wildcard-val.bess go into bessctl/module_tests to keep conf folder minimal?